packages/core/src/components/popover/popover.tsx

```diff
@@ -52,12 +52,6 @@ export interface IPopoverProps extends IOverlayableProps, IProps {
     arrowSize?: number;
 
     /**
-     * Constraints for the underlying Tether instance.
-     * See http://tether.io/#constraints.
-     */
-    constraints?: TetherUtils.ITetherConstraint[];
```

Comment:
🔥  unfortunately we cannot remove props until `v2.0`.  please mark this as `@deprecated` in the comment, but you must continue to support it alongside `tetherOptions`.